Mathlib/Analysis/RCLike/Basic.lean

```diff
@@ -902,6 +902,18 @@ lemma ofReal_pos {x : ℝ} : 0 < (x : K) ↔ 0 < x := by
 lemma ofReal_lt_zero {x : ℝ} : (x : K) < 0 ↔ x < 0 := by
   rw [← ofReal_zero, ofReal_lt_ofReal]
 
+lemma inv_pos_of_pos (hz : 0 < z) : 0 < z⁻¹ := by
+  rw [pos_iff_exists_ofReal] at hz
+  obtain ⟨x, hx, hx'⟩ := hz
+  rw [← hx', ← ofReal_inv, ofReal_pos]
+  exact _root_.inv_pos_of_pos hx
+
+lemma inv_pos : 0 < z⁻¹ ↔ 0 < z := by
```

Comment:
Probably this and `inv_pos_of_pos` should be protected? That way we can avoid the `_root_.inv_pos` bits. If you really think it won't come up that often, you can leave it and mark this as resolved.

Reply:
Fair point; I've made them protected.